track.go

```diff
@@ -355,6 +392,7 @@ func newVideoTrackFromReader(source Source, reader video.Reader, selector *Codec
 		if err != nil {
 			base.onError(err)
 		}
+		base.onImgRead(img, err)
```

Comment:
should we be capturing the second output from Read? 
(i presume it's the number of bytes read as is typical with readers)

Reply:
you would be incorrect that is a release function, there is no indication of the number of bytes read ever.